pyblish_standalone/__main__.py

```diff
@@ -22,6 +22,11 @@ def cli():
                               "can be called multiple times"))
     parser.add_argument("-rh", "--register-host", action="append",
                         help=("Append hosts to register."))
+    parser.add_argument("-rg", "--register-gui", action="append",
+                        help=("Append guis to register."))
+    parser.add_argument("--debug", action="store_true")
+    parser.add_argument("--port", nargs='?',
```

Comment:
Sorry, I didn't realise I had a PR going.

It was to make the ```---port``` flag optional, but I might end up removing it and use environment variables to pass the port number to pyblish_aftereffects.